components/registry/org.wso2.carbon.registry.search.ui/src/main/resources/web/search/js/search.js

```diff
@@ -226,10 +226,10 @@ function initMiscFields() {
     }
 }
 
-function submitAdvSearchForm(pageNumber) {		
+function submitAdvSearchForm(pageNumber) {
```

Comment:
There were unnecessary tab spacing were included.  It was removed by the formatting. Hence the formatting is correct